src/components/HeaderHome.js

```diff
@@ -0,0 +1,24 @@
+import {useDispatch, useSelector} from "react-redux";
+import {openModal} from "../store/modalStatusSlice";
+
+const HeaderHome = () => {
+
+    const invoices = useSelector(state => state.invoices.entities)
+    const invoicesNumber = Object.keys(invoices).length;
+
+    const dispatch = useDispatch()
+    const openNewInvoice = () => {
+        dispatch(openModal());
+    }
+
+    return (
+        <div>
+            <h1>Invoices</h1>
+            <h6>There are {invoicesNumber} total invoices</h6>
```

Comment:
Everything seems to be great here. I have only some small suggestions: I think this is better to use here <p> tag instead of <h6>. Concerning using headings we should keep the right order of them - after h1 it should be h2 etc. A good article about headings is here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements. For future todo (I think that this is a good task for me - we should think about some conditionals concering 2 cases: ("No invoices" and "There is 1 invoice"). Also in mobile there is a shorter text.